website/content/en/preview/faq.md

```diff
@@ -176,10 +176,10 @@ Yes, see the [KubeletConfiguration Section in the NodePool docs]({{<ref "./conce
 The difference between the Core and Full variants is that Core is a minimal OS with less components and no graphic user interface (GUI) or desktop experience.
 `Windows2019` and `Windows2022` AMI families use the Windows Server Core option for simplicity, but if required, you can specify a custom AMI to run Windows Server Full.
 
-You can specify the [Amazon EKS optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html) with Windows Server 2022 Full for Kubernetes 1.27 by configuring an `amiSelector` that references the AMI name.
+You can specify the [Amazon EKS optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html) with Windows Server 2022 Full for Kubernetes 1.28 by configuring an `amiSelector` that references the AMI name.
 ```
-amiSelector:
-    aws::name: Windows_Server-2022-English-Full-EKS_Optimized-1.27*
+amiSelectorTerms:
+    - name: Windows_Server-2022-English-Full-EKS_Optimized-1.28*
```

Comment:
Should we just make this a general version? We wouldn't have to update it and it would take the latest AMI anyway.
```suggestion
    - name: Windows_Server-2022-English-Full-EKS_Optimized-*
```